database/manual-updates/2021-06-14_1-add-personal-relationships-tables.md

```diff
@@ -0,0 +1,66 @@
+# Add personal relationships tables
+
+## The problem we're trying to solve
+
+We need to add three new tables to store personal relationships.
+
+## Justification for doing a manual update
+
+We don't have database migrations set up for the API.
+
+## The plan
+
+1. Run SQL statements to create the tables in Staging
+2. Run SQL statements to create the tables in Production
+
+## Link to Jira ticket
+
+https://hackney.atlassian.net/browse/SCT-409
+
+## SQL statement(s)
+
+```sql
+CREATE TABLE IF NOT EXISTS dbo.sccv_personal_relationship_type (
+    id bigint GENERATED BY DEFAULT AS IDENTITY NOT NULL CONSTRAINT xpk_personal_relationship_type PRIMARY KEY,
+    description varchar(300),
+    inverse_type_id bigint
+);
+
+CREATE TABLE IF NOT EXISTS dbo.sccv_personal_relationship (
+    id bigint GENERATED BY DEFAULT AS IDENTITY NOT NULL CONSTRAINT xpk_personal_relationship PRIMARY KEY,
+    fk_person_id bigint,
+    fk_second_person_id bigint,
```

Comment:
Or `fk_other_person_id`?

Reply:
fk_other_person_id sounds a bit better, up to you if you want to change it or not :D

Reply:
Will make the change! 😝